### DISCUSSION TITLE
Funds sent from OKX exchange dont appear in the wallet

### DISCUSSION THREAD

**Discussion by anarhaifa** [Chia-Network/chia-blockchain#9955] [Support]
Hey guys,help please
Txid - https://www.chiaexplorer.com/tx/0x4e672ada9dfab1c6a6d63b2a3a6bfa2f1dbf77cbffa34b458f386dcb7d68a2b2

**Comment by minerfloripa** [comment]
same problem here.

**Reply by anarhaifa** [reply]
the funds do appear when i put my wallet address here - https://www.chiaexplorer.com/blockchain/search
not in the wallet dashboard though

**Comment by anarhaifa** [comment]
Support guys,you out of the office or what ?

**Comment by anarhaifa** [comment]
Great support guys. 25 days total ignore.

**Reply by anarhaifa** [reply]
Anyone ????

**Comment by anarhaifa** [comment]
Day 28

**Comment by anarhaifa** [comment]
1 month anniversery,to support ignoring my request !

**Comment by anarhaifa** [comment]
Heeeeeeeeeelp

**Comment by anarhaifa** [comment]
S.O.S

**Comment by esaung** [comment]
which version of the wallet are you running?

**Reply by anarhaifa** [reply]
1.2.11

**Comment by anarhaifa** [comment]
CHIA SUPPORT WHERE YA AT

**Comment by esaung** [comment]
try on a different machine to use our 1.3 beta here - https://www.chia.net/download/

choose wallet mode (for the faster sync time) - and let me know if you see your XCH

**Reply by anarhaifa** [reply]
Different machine you mean another PC ?

**Comment by esaung** [comment]
yes, different computer

**Reply by anarhaifa** [reply]
Tried but nothing

**Comment by anarhaifa** [comment]
Any other ways to solve my problem ?

**Comment by anarhaifa** [comment]
Support,how about doing some supporting ?

**Comment by anarhaifa** [comment]
Still no mulla in my wallet

**Reply by loppefaaret** [reply]
can you confirm that your node is fully synced, and the wallet is showing fully synced as well ?
are you currently on 1.2.11 or 1.3 ? 
this is the address you are referring to `xch12hg7rwaty7xqwup6pfunll8pjqnkt8mxgjqyrlhjzkzrg3jwag0qvqm32p` ?

**Reply by anarhaifa** [reply]
Versiion 1.2.11. Status Not Synced . Address you wrote is correct

**Reply by anarhaifa** [reply]
any thouhgts ?

**Reply by loppefaaret** [reply]
If it isnt synced, im thinking there is a very good chance that you have yet to pass the block that contains the transaction, and therefor are not seeing the coins yet - i would recommend trying to install the 1.3.0 release, and run it in "Wallet only" mode.
I feel confident that withing minutes, you will see and have control over your funds

**Reply by anarhaifa** [reply]
thx man,ill update if there's any change

**Reply by anarhaifa** [reply]
I see the funds on my balance thanks alot for helping !
